src/database/postgres/create_tables.md

```diff
@@ -0,0 +1,31 @@
+## Create tables in a Postgres database
+
+[![postgres-badge]][postgres] [![cat-database-badge]][cat-database]
+
+Use the `postgres` crate to create tables in a Postgres database. 
+
+The recipe assumes an existing database named `library`, the username is `postgres` and the password is `postgres`.
+
+```rust,no_run
+extern crate postgres;
+
+use postgres::{Connection, TlsMode};
+
+fn main() {
+    let conn = Connection::connect("postgresql://postgres:postgres@localhost/library", 
+                                    TlsMode::None).unwrap();
```

Comment:
Should use `?` instead of `unwrap()` throughout the examples

Reply:
👍 This will include making the main function return a result FYI

Reply:
@AndyGauge I am trying to figure this out. 

The `postgres` crate also defines Result and Error types. 
I tried -

`fn main() -> Result<(), std::io::Error>`. 

But this gives an error - 

`_^ expected enum std::result::Result, found ()`.

Reply:
@AndyGauge I think I found the solution. I need to use `Ok(())` at the end of main.

Reply:
>  This will include making the main function return a result FYI

Done. Please review.